homeassistant/components/brother/const.py

```diff
@@ -165,6 +165,6 @@
     ATTR_UPTIME: {
         ATTR_ICON: "mdi:timer-outline",
```

Comment:
When setting a device class, one should not set an icon.
So either remove the device class or the icon.

Reply:
I forgot about that. Fixed.